client/src/components/Header.jsx

```diff
@@ -1,81 +1,36 @@
-import React from 'react';
 import SearchAndFilter from './SearchAndFilter';
-//import { jwtDecode } from 'jwt-decode';
-import { useGoogleLogin, googleLogout } from '@react-oauth/google';
-import {
-  Dropdown,
-  DropdownButton,
-  Navbar,
-  Container,
-  Button,
-} from 'react-bootstrap';
+import { useGoogleLogin } from '@react-oauth/google';
+import { Dropdown, Navbar, Container, Button } from 'react-bootstrap';
 import { useNavigate } from 'react-router-dom';
+import useAuthStore from '../store/authStore';
```

Comment:
What's the standard way to order imports? can we put the {} ones together and the not- {} ones together?

Reply:
resolved